SENG-499-Company2-B01/Backend
# C2 2 mongo db

# Initial MongoDB Container 

In this ticket I created a mongodb container which creates a basic mongodb instance for the backend service. I have also created 2 simple bash scripts which automates the build / tear down of the mongo container for ease of use.   

## Files of Interest (under mongodb directory)
1. Dockerfile 
2. build_mongo.sh 
3. stop_mongo.sh

## Steps to Demo / Test  

1. Start by running: 
`bash build_mongo.sh` 

4. Then verify the container is running and in a healthy state: 
`docker ps -a` 

5. Verify that you can login to the database: 
`docker exec -it mongodb bash` , then once you are in the container run `mongo -u "admin" -p "admin"` and ensure the login was successful 

6. Stop Mongo by running: 
`bash stop_mongo.sh` , then run `docker ps -a` to verify the container has indeed been removed
